include/multipass/memory_size.h

```diff
@@ -32,7 +32,7 @@ class MemorySize
     friend bool operator<=(const MemorySize& a, const MemorySize& b);
     friend bool operator>=(const MemorySize& a, const MemorySize& b);
 
-    MemorySize();
+    MemorySize() noexcept;
```

Comment:
Ok, just to make sure I understand, the fact that setting the bytes to 0 is a non-throwing operation, marking this `noexcept` is telling the reader and compiler explicitly that there is no way this will throw, right?

Reply:
Exactly. That's the recommendation in such cases. Note that this propagates to callers: a caller that calls only this and other `noexcept` methods can now declare itself `noexcept`. A bit like marking methods that don't change an object as const.

Reply:
Ok, cool :+1: